frontend/public/components/storage-class.tsx

```diff
@@ -18,23 +18,19 @@ const StorageClassHeader = props => <ListHeader>
   <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
   <ColHead {...props} className="col-sm-4 col-xs-6" sortField="provisioner">Provisioner</ColHead>
   <ColHead {...props} className="col-sm-2 hidden-xs" sortField="reclaimPolicy">Reclaim <span className="hidden-sm">Policy</span></ColHead>
-  <ColHead {...props} className="col-sm-2 hidden-xs" sortField={`metadata.annotations['${defaultClassAnnotation}']`}>Default <span className="hidden-sm">Class</span></ColHead>
 </ListHeader>;
 
 const StorageClassRow: React.SFC<StorageClassRowProps> = ({obj}) => {
   return <div className="row co-resource-list__item">
     <div className="col-sm-4 col-xs-6 co-break-word">
-      <ResourceLink kind={StorageClassReference} name={obj.metadata.name} namespace={undefined} title={obj.metadata.name} />
+    <React.Fragment> <ResourceLink inline={true} kind={StorageClassReference} name={obj.metadata.name} namespace={undefined} title={obj.metadata.name} /> { isDefaultClass(obj) == 'true' ? '- Default' : null } </React.Fragment>
```

Comment:
Please break this onto multiple lines.

Use `&ndash;` entity for the dash.

Prefer `===` to `==`.

We should change `isDefaultClass` to return a boolean instead of a string if we're not using it as a dispaly value.

This doesn't have the

Reply:
Also I believe `React.Fragment` is unnecessary here

Reply:
Ack. Will look into all changes suggested.

Reply:
> We should change `isDefaultClass` to return a boolean instead of a string if we're not using it as a dispaly value.

`isDefaultClass` is still used in the Storage Class Overview page and the string true/false is displayed there.

Reply:
Right, change the helper to return a boolean, then show `true` or `false` strings on the details page. If any of the callers need to check it as a boolean, better to have the helper return a boolean.

Reply:
Okay sure, I'll do that :) 
Also, is `&ndash` something that is imported? It's not working the way I thought it would, am I using it inncorrectly?

Reply:
> Also, is `&ndash` something that is imported? It's not working the way I thought it would, am I using it inncorrectly?

It's an HTML entity. Use `&ndash;` in the markup

Reply:
Alright. Thanks!